kernel/transparentState.mli

```diff
@@ -28,7 +29,11 @@ val var_full : t
 val cst_full : t
 (** All constant transparent *)
 
+val prj_full : t
```

Comment:
Not sure what the intent of these x_full values is. AFAICT there is only 1 user of cst_full (in rewrite) and 1 of var_full (in unification), IMO I would remove them from transparentstate.ml and have them next to the unique user.
Also the user of cst_full probably wants projections to be transparent too.